pkgs/os-specific/linux/ati-drivers/default.nix

```diff
@@ -77,7 +77,10 @@ stdenv.mkDerivation rec {
   ++ optionals ( kernel != null &&
                  (builtins.compareVersions kernel.version "4.6") >= 0 )
                [ ./patches/kernel-4.6-get_user_pages.patch
-                 ./patches/kernel-4.6-page_cache_release-put_page.patch ];
+                 ./patches/kernel-4.6-page_cache_release-put_page.patch ]
+  ++ optionals ( kernel != null &&
+                 (builtins.compareVersions kernel.version "4.7") >= 0 )
```

Comment:
I thought I looked for something like that when I did the 4.6 patches and didn't find it.  I'll try to revise tonight.